app/Console/Commands/Members/ImportMembers.php

```diff
@@ -113,24 +144,28 @@ protected function createNewMember($member_data)
         // if they have an extra rating, log their previous rating first,
         // regardless of whether it will be overwritten
         if ($member_data['rating_atc'] >= 8) {
-            try {
-                $_prevRat = VatsimXML::getData($member->id, 'idstatusprat');
-            } catch (Exception $e) {
-                if (strpos($e->getMessage(), 'Name or service not known') !== false) {
-                    // CERT unavailable. Not our fault, so will ignore.
-                    return;
-                }
-
-                return;
-            }
-
-            if (isset($_prevRat->PreviousRatingInt)) {
-                $prevAtcRating = Qualification::parseVatsimATCQualification($_prevRat->PreviousRatingInt);
-
-                if ($prevAtcRating) {
-                    $member->addQualification($prevAtcRating);
-                }
-            }
+            // This user has an admin rating but there is currently no support
```

Comment:
This worries me a little

Reply:
I'm not convinced prat has been working anyway to be fair. I've raised it to .net to consider for a future update.